config/samples/ramendr_v1alpha1_volumereplicationgroup.yaml

```diff
@@ -4,8 +4,7 @@ metadata:
   name: volumereplicationgroup-sample-1
 spec:
   # Add fields here
-  affinedCluster: us-central
-  applicationLabels:
+  pvcLabelSelector:
     matchExpressions:
     - key: apptype
```

Comment:
Can we remove the `matchExpressions` from the sample? We only need the labels, and was interfering in the tests. I almost thought the PVC watcher was not working :) but realized my PVCs only had the labels and no expressions to match.

As the sample is modified in this PR, requesting removal here.